test/unit/api/ingest/_data.js

```diff
@@ -0,0 +1,193 @@
+define(function (require) {
+  var Promise = require('bluebird');
+  var _ = require('intern/dojo/node!lodash');
+  var expect = require('intern/dojo/node!expect.js');
+  var fakeNamesIndexTemplate = require('intern/dojo/node!../../data/fake_names_index_template.json');
```

Comment:
Nit: consider naming the unit test `data` directory as `fixtures` as that conforms to existing vocabulary around testing.